addons/website/models/ir_ui_view.py

```diff
@@ -270,6 +270,15 @@ def get_inheriting_views_arch(self, model):
         # prefer inactive website-specific views over active generic ones
         return views.filter_duplicate().filtered('active')
 
+    @api.model
+    def combine_with_specific_views(self, views, ids_to_check):
```

Comment:
I think arguments should be:

views_ids, unfiltered_views_ids

or

views, unfiltered_views

but not mixing ids and not ids in arguments name